test/integration/http_timeout_integration_test.cc

```diff
@@ -95,4 +95,145 @@ TEST_P(HttpTimeoutIntegrationTest, PerTryTimeout) {
   EXPECT_EQ("504", response->headers().Status()->value().getStringView());
 }
 
+// With hedge_on_per_try_timeout enabled via config, sends a request with a
+// global timeout and per try timeout specified, sleeps for longer than the per
+// try but slightly less than the global timeout. We then have the first
+// upstream request return headers and expect those to be returned downstream
+// (which proves the request was not canceled when the timeout was hit).
+TEST_P(HttpTimeoutIntegrationTest, HedgedPerTryTimeout) {
```

Comment:
Can you make sure to run these integration tests with runs_per_test set to 1000+ to look for flakes?

Reply:
ran it with 1000 runs per test and didn't get any flakes